src/main/electron/listener.cljs

```diff
@@ -156,27 +156,35 @@
 
   (js/window.apis.on "quickCapture"
                      (fn [args]
-                       (let [{:keys [url title content]} (bean/->clj args)
-                             insert-today? (get-in (state/get-config)
-                                                   [:quick-capture-options :insert-today]
-                                                   false)
-                             page (if (true? insert-today?) (string/lower-case (date/journal-name)) (state/get-current-page))
```

Comment:
@hkgnp `(state/get-current-page)` might be a `nil` on journal pages. So quickCapture fails.

Never mind, A config option can be added.

Reply:
I see! Good catch!

I am actually personally ok with the settings change outside Logseq. It just means I will have to update my extensions and include a settings page. 

Know you're busy with other stuff already so let me know what works best for you!

Reply:
Since it's not released yet. A lot can be done.
I'll add `[:quick-capture-options :insert-today]` option back, as the fallback of URL parameters.